charts/vh-video-api/values.yaml

```diff
@@ -83,7 +83,10 @@ java:
         - vodafoneconfiguration--pexipnode
         - vodafoneconfiguration--pexipselftestnode     
         - vodafoneconfiguration--callbacksecret   
-        - vodafoneconfiguration--telephoneconferencenumber
+        - name: vodafoneconfiguration--telephoneconferencenumber
+          alias: vodafoneconfiguration--conferencephonenumber
+        - name: vodafoneconfiguration--telephoneconferencenumberwelsh
+          alias: vodafoneconfiguration--conferencephonenumberwelsh
         - vodafoneconfiguration--telephoneconferencenumberwelsh
```

Comment:
telephoneconferencenumberwelsh is present twice. please remove the one we don't need

Reply:
done

Reply:
done